github/github.go

```diff
@@ -97,6 +97,9 @@ const (
 
 	// https://developer.github.com/changes/2017-02-09-community-health/
 	mediaTypeRepositoryCommunityHealthMetricsPreview = "application/vnd.github.black-panther-preview+json"
+
+	// https://developer.github.com/v3/codes_of_conduct/
```

Comment:
This link isn't great, because the mention of preview API will eventually be removed.

Let's link to https://developer.github.com/changes/2017-05-23-coc-api/ here instead.